memorious/operations/db.py

```diff
@@ -4,17 +4,49 @@
 from memorious.core import datastore
 
 
-def db(context, data):
-    """Insert or update `data` as a row into specified db table"""
-    table = context.params.get("table", context.crawler.name)
+def _recursive_upsert(context, params, data):
+    """Insert or update nested dicts recursively"""
+    table = params.get("table")
     table = datastore.get_table(table, primary_id=False)
-    unique_keys = ensure_list(context.params.get("unique"))
-    # context.log.info("Save: %s %r", table.name, unique_keys)
+    unique_keys = ensure_list(params.get("unique"))
+
+    children = params.get("children", {})
+    nested_calls = []
+    for child_params in children:
+        key = child_params.get("key")
+        child_data_list = data.pop(key, [])
+        if isinstance(child_data_list, dict):
+            child_data_list = [child_data_list]
+        if not (isinstance(child_data_list, list) and
+                all(isinstance(i, dict) for i in child_data_list)):
+            context.log.warn(
+                "Expecting a dict or a lost of dicts as children for key", key
+            )
+            continue
+        if child_data_list:
+            table_suffix = child_params.get("table_suffix", key)
+            child_params["table"] = params.get("table") + "_" + table_suffix
+            parent_id = data.get(child_params.get("parent_id"))
+            for child_data in child_data_list:
+                child_data["__parent_id"] = parent_id
```

Comment:
I think we should make that field name configurable, while setting a default. We often want it to be something like "company ID" or so, which would be nice to have as a proper field name.